.babelrc

```diff
@@ -6,7 +6,14 @@
         "react",
         "stage-2"
       ],
-      "plugins": ["babel-plugin-rewire"]
+      "plugins": [
+        "babel-plugin-rewire",
+        ["module-resolver", {
+          "alias": {
+            "preact-compat": "react"
```

Comment:
Why do we alias if we're using preact directly?

Reply:
But the real output that the user will have is the one from preact-compat right? So might as well use this one in the snapshots?

Reply:
I need to take at least half a day to update theses snapshots, wan't to make sure everything is working for every widgets, let's plan this?

Reply:
https://github.com/algolia/instantsearch.js/issues/2405 ok